jujugui/static/gui/src/app/init.js

```diff
@@ -1658,6 +1660,10 @@ class GUIApp {
     // Fetch the url for the config file from the DOM. This is necessary because
     // the config path is defined on render by the various deployment environments.
     const oldScript = doc.querySelector('script[data=config]');
+    if (!oldScript) {
+      // We must be using Parcel so no need to do anything.
```

Comment:
Is this true? Don't we still need to get the config values and refresh them to merge in changes supplied by the user?